apps/cow-fi/components/ArticlesPageComponents.tsx

```diff
@@ -97,8 +96,8 @@ export function ArticlesPageComponents({
               <h1>All articles</h1>
             </Breadcrumbs>
             <ArticleCount>
-              Showing {ITEMS_PER_PAGE * (currentPage - 1) + 1}-{Math.min(ITEMS_PER_PAGE * currentPage, totalArticles)}{' '}
-              of {totalArticles} articles
+              Showing {ARTICLES_PER_PAGE * (currentPage - 1) + 1}-
+              {Math.min(ARTICLES_PER_PAGE * currentPage, totalArticles)} of {totalArticles} articles
```

Comment:
we have a bunch of pagination math inside the content. Their place should be some utils. We also have a bit above the one that gets you the number of pages

Reply:
Addressed in https://github.com/cowprotocol/cowswap/pull/5655 and merged in this release branch.